include/pipettes/core/pipette_messages.hpp

```diff
@@ -32,14 +35,17 @@ struct GetStatusResult {
     uint32_t data;
 };
 
-struct Status {};
+struct SetDistance {
+    uint32_t mm;
```

Comment:
Whats the difference between "set distance" and "move"?

Reply:
Move replaces set distance, so we can remove that

Reply:
Yeah, I was planning to refactor that when we flesh out the move command.